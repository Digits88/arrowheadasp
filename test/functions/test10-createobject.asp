<!-- Test CreateObject function.
-->

Dim A <% Dim A %><br>
Set A = CreateObject("Scripting.Dictionary")
    <% Set A = CreateObject("Scripting.Dictionary") %><br>
A("Hello") = "there" 
    <% A("Hello") = "there" %><br>
A("Hello") = <%= A("Hello") %><br>
