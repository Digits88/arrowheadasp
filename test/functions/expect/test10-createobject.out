<!-- Test CreateObject function.
-->

Dim A <br>
Set A = CreateObject("Scripting.Dictionary")
    <br>
A("Hello") = "there" 
    <br>
A("Hello") = there<br>
